api/graphql/schema.graphql

```diff
@@ -283,6 +284,15 @@ type UserPublicProfile @model(class: "\\App\\Models\\User") {
   lastName: String @rename(attribute: "last_name")
 }
 
+type BasicGovEmployeeDetails @model(class: "\\App\\Models\\User") {
```

Comment:
Shoot, seeing this now, I see that it's confusing that it sounds like a plural name.  Maybe `BasicGovEmployeeProfile` so it aligns with the UserPublicProfile and it sounds singular?

Reply:
Good call, renamed in [rename type and query](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12847/commits/0c60b716655f506fa9f05ed90e2614f00be3f783)